index.js

```diff
@@ -144,15 +144,7 @@ function getErrorHeaders (err) {
     return undefined
   }
 
-  var headers = Object.create(null)
-  var keys = Object.keys(err.headers)
-
-  for (var i = 0; i < keys.length; i++) {
-    var key = keys[i]
-    headers[key] = err.headers[key]
-  }
-
-  return headers
+  return { ...err.headers }
```

Comment:
This is functionally different than the previous because this object does not have a null prototype. There are a few ways we could fix it, but I am not sure what would be best.

Reply:
I don't think we need the null prototype here because the only usage of the headers is to iterate over them and set them on the response. I think we don't even need to copy the headers we can iterate over them directly.

Reply:
The null prototype helps protect against some forms of mistakes consumers of the package can make which lead to security issues, so it is important to be vigilant on this type of change. BUT! Now that I have had a second to look at it, I think you are correct because in this case the method is not exported *and* we are only using the return value within this package. I *think* that protects from all the possible mistakes end users could make.

Reply:
Also, just be be sure, I went through the history of this a bit, links for reference:

https://github.com/pillarjs/finalhandler/commit/7bfa2f71fbe2b7f3df465ca6748e0dbc68a91cdf
https://github.com/pillarjs/finalhandler/commit/ab539c3a148532ca621f05d0f96653905d3071c8

I am guessing this was done out of an abundance of caution back then. I don't see a way this could become a security issue even in the original form. Maybe I am missing something, but I am again thinking this is safe to remove.